src/main/java/reposense/RepoSense.java

```diff
@@ -110,6 +115,8 @@ public static void main(String[] args) {
             FileUtil.zipFoldersAndFiles(reportFoldersAndFiles, cliArguments.getOutputFilePath().toAbsolutePath(),
                     ".json");
 
+            GitConfig.setGlobalGitLfsConfig(globalGitConfig);
```

Comment:
Let's add a comment before this line to explain that we are setting the git config back to the original value, so that it is clearer for future developers.